### ISSUE TITLE
the fn4_shutdown.sh script can leave processes running

### ISSUE THREAD

**Issue by davidhwyllie** [davidhwyllie/findNeighbour4#113]
Team of @jodugo reported that the fn4_shutdown.sh script can leave processes running.  It is reported this can leave locks open, requiring manual closure.

Testing performed.

Identified one process left open reproducibly by current code:
* findNeighbour4_lockmanager    

Additionally, it is possible that the following may be left open although I cannot reproduce this:
* findNeighbour4_clustering.py

Examples from ps x | grep covid_config # covid_config is part of the config json file name 

e.g.
```
python3 findNeighbour4_lockmanager.py --path_to_config_file demos/covid/covid_config_rdbms.json
python3 findNeighbour4_clustering.py demos/covid/covid_config_rdbms.json
```

**Comment by davidhwyllie** [comment]
The patterns present in fn4_shutdown.sh do not shutdown findNeighbour4_lockmanager.py with its current command line syntax.
Modified fn4_shutdown.sh to address this.

**Comment by davidhwyllie** [comment]
Resolved by PR #115
